src/elements/content-preview/preview-header/PreviewHeader.js

```diff
@@ -100,7 +101,13 @@ const PreviewHeader = ({
                                     {...contentOpenWithProps}
                                 />
                             )}
-                            {shouldRenderAnswers && <ContentAnswers />}
+                            {shouldRenderAnswers && (
+                                <ContentAnswers
+                                    fileExtension={currentExtension}
```

Comment:
Are there any tests for PreviewHeader that we need to update after adding these props?

Reply:
updated